src/components/slider/HomePageTopSlider.vue

```diff
@@ -0,0 +1,123 @@
+<template>
+    <div>
+        <carousel :per-page="1" :paginationActiveColor="'#00aeb0'" :autoplay="true" :autoplayTimeout="1500" :loop="true">
```

Comment:
if you have more than 2 attributes in html tag use next syntax (in this example):
<carousel 
  :per-page="1"
  :paginationActiveColor="'#00aeb0'" 
  :autoplay="true"
  :autoplayTimeout="1500"
  :loop="true"
>